<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SereneUClass</class>
 <widget class="QMainWindow" name="SereneUClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SereneU</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>예약관리</string>
     </attribute>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>10</y>
        <width>571</width>
        <height>231</height>
       </rect>
      </property>
      <property name="gridVisible">
       <bool>true</bool>
      </property>
      <property name="verticalHeaderFormat">
       <enum>QCalendarWidget::VerticalHeaderFormat::NoVerticalHeader</enum>
      </property>
      <property name="navigationBarVisible">
       <bool>true</bool>
      </property>
      <property name="dateEditEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableView" name="reservationView">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>250</y>
        <width>571</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addCustomerButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>490</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>고객추가</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addReservationButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>490</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>예약추가</string>
      </property>
     </widget>
     <widget class="QLabel" name="dbConnect">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>500</y>
        <width>301</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>DB연결끼기기</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>매출관리</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>고객관리</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>481</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="customerName111_2">
         <property name="text">
          <string>이름</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="customerName_2"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>전화번호</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="customerPhone_2"/>
       </item>
       <item>
        <widget class="QPushButton" name="searchCustomer_2">
         <property name="text">
          <string>조회</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableView" name="customerView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>481</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>618</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDB">
    <property name="title">
     <string>DB</string>
    </property>
    <addaction name="actionDB"/>
   </widget>
   <addaction name="menuDB"/>
  </widget>
  <action name="actiontest">
   <property name="text">
    <string>시수목록추가</string>
   </property>
  </action>
  <action name="dbConnector">
   <property name="text">
    <string>연결관리</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>지출내역</string>
   </property>
  </action>
  <action name="actionDB">
   <property name="text">
    <string>DB관리</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
